packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -28,7 +28,7 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
         <input
           type="checkbox"
           className={styles.input}
-          checked={value}
+          checked={value ?? false}
```

Comment:
This will break uncontrolled checkboxes (which we use quite a lot inside forms) since the checkbox will always have a value and passing `defaultChecked` to it won't work anymore (an example can be seen in the storybook in uncontrolled story).